contrib/docker/nightly-tester/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:22.04
+
+LABEL maintainer <rene.gassmoeller@mailbox.org>
+
+RUN DEBIAN_FRONTEND=noninteractive apt update && apt upgrade -yq && \
+  DEBIAN_FRONTEND=noninteractive apt install -yq --no-install-recommends \
+  build-essential ca-certificates file gcc g++ \
+  gfortran git libblas-dev liblapack-dev libopenmpi-dev \
+  lsb-release ninja-build numdiff openmpi-bin \
+  openmpi-common wget zlib1g-dev \
+  texlive-plain-generic texlive-base texlive-latex-recommended \
+  texlive-latex-base texlive-fonts-recommended \
+  texlive-bibtex-extra lmodern texlive-latex-extra \
+  texlive-science graphviz python3-pip python-setuptools \
+  clang-tidy clang cmake libp4est-dev libsundials-dev \
+  mpi-default-dev trilinos-all-dev zlib1g-dev libhdf5-dev libhdf5-mpi-dev
+
+COPY local.cfg /opt/
+
+# Install deal.II with minimal dependencies for ASPECT
+
+RUN cd /opt && \
+    git clone https://github.com/dealii/candi && \
+    cd candi && \
+    mv /opt/local.cfg . && \
+    CXX=clang++ CC=clang ./candi.sh -p /opt -j4 && \
```

Comment:
not sure -j4 will work. The github actions testers have 2 cores, and only limited RAM (maybe 8GB?)